index.html

```diff
@@ -0,0 +1,191 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Youtube-Clone</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+</head>
+
+<body>
+    <nav>
+        <div class="left-icons">
+            <a href="" class="menu-icon">
+                <span>
+                    <div class="bar"></div>
+                    <div class="bar"></div>
+                    <div class="bar"></div>
+                </span>
+            </a>
+            <span>
+                <a hred="">
+                    <img src="Essentials/yt_logo_rgb_light.svg" id="yt-logo">
+                </a>
+            </span>
+        </div>
+        <div class="search">
+            <input type="search" id="searchbar" value="Search">
+            <div class="src-btn">
+                <div id="magnifying-glass"></div>
+            </div>
+        </div>
+        <div class="right-icons">
```

Comment:
Great on implementing list items in a nav, but it's better to use semantic unorder list ul for all the listed items inside the nav for better code readability, user experience, and greater accessibility. Kindly use a ul tag instead of a nav.